### PR TITLE
Update paths to renci python image

### PR DESCRIPTION
**Pull request by maximusunc**

### CODE REVIEW

**Review by kennethmorton — APPROVED** [review]
LGTM!

File: strider/trapi.py [hunk 1]
```diff
@@ -492,6 +498,16 @@ def convert_subclasses_to_aux_graphs(
...
+                    message.knowledge_graph.edges[primary_edge_id].attributes.add(
+                        Attribute.parse_obj(
+                            {
+                                "attribute_type_id": "biolink:knowledge_level",
+                                "value": "logical_entailment",
+                                "attribute_source": "infores:aragorn",
+                            }
+                        )
+                    )
+
```
Comment (uhbrar) [inline strider/trapi.py:510]: We need to also remove the existing `knowledge_level` attribute from this edge that would have been created by the KP.

**Review by uhbrar — CHANGES_REQUESTED** [review]

File: strider/trapi.py [hunk 2]
```diff
@@ -452,7 +452,13 @@ def convert_subclasses_to_aux_graphs(
...
                             "object": node.id,
                             "predicate": "biolink:subclass_of",
-                            "attributes": [],
+                            "attributes": [
+                                {
+                                    "attribute_type_id": "biolink:knowledge_level",
+                                    "value": "not_provided",
+                                    "attribute_source": "infores:aragorn",
+                                },
+                            ],
```
[the remark this answers is not in the document]
Comment (maximusunc) [reply strider/trapi.py]: Added.

File: strider/trapi.py (hunk 1 above)
Comment (maximusunc) [reply strider/trapi.py]: Now we'll overwrite knowledge_level if it's present, otherwise add a new attribute

**Review by uhbrar — APPROVED** [review]
This looks good now. KL/AT are both required attributes, so if they are not present for an edge when receiving it from a KP, we should probably log it and set it to `not_provided` so that we can then post an issue for that KP. That's beyond the scope of this PR though.

**Pull request merged**